src/mono/mono/mini/interp/transform.c

```diff
@@ -6889,7 +6947,7 @@ generate_code (TransformData *td, MonoMethod *method, MonoMethodHeader *header,
 				default: {
 					GString *res = g_string_new ("");
 					mono_type_get_desc (res, m_class_get_byval_arg (klass), TRUE);
-					g_print ("STELEM: %s -> %d (%s)\n", m_class_get_name (klass), mint_type (m_class_get_byval_arg (klass)), res->str);
+					g_print ("STELEM: %s -> %d (%s)\n", m_class_get_name (klass), mono_mint_type (m_class_get_byval_arg (klass)), res->str);
```

Comment:
Should this g_print be guarded by a td->verbose check or something?

Reply:
this is assertion code